conda_smithy/configure_feedstock.py

```diff
@@ -1836,6 +1904,10 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
         },
         "github_actions": {
             "self_hosted": False,
+            "self_hosted_labels": [],
```

Comment:
Can this be done per build_platform? We would need different labels for different providers like cirun and win-arm64 workers.

Reply:
Oh, yes, I see what you mean. This would also help if we need to use the GPUs only on Linux-64!